static/js/components/ManageDataForm.jsx

```diff
@@ -136,13 +137,21 @@ const useStyles = makeStyles(() => ({
   },
 }));
 
-const SpectrumRow = ({ rowData, route }) => {
+const SpectrumRow = ({ rowData, route, annotations }) => {
   const styles = useSourceStyles();
   const colSpan = rowData.length + 1;
+  const spectrumID =
+    typeof rowData[0] === "number" ? rowData[0] : parseInt(rowData[0], 10);
```

Comment:
```suggestion
  const spectrumID = parseInt(rowData[0], 10);
```
no harm in passing in a value that's already an int; this is much easier to read/maintain.